docs/api-reference/widgets/scale-widget.md

```diff
@@ -0,0 +1,61 @@
+import {WidgetPreview} from '@site/src/doc-demos/widgets';
+import {_ScaleWidget} from '@deck.gl/widgets';
+
+# ScaleWidget (Experimental)
+
+<img src="https://img.shields.io/badge/from-v9.2-green.svg?style=flat-square" alt="from v9.2" />
+
+This widget displays a dynamic scale bar (like Google Maps) that updates as the map zoom and latitude change. It renders  a horizontal line with two tick marks at the ends, and a formatted distance label.
+
+<WidgetPreview cls={_ScaleWidget}/>
+
+```ts
+import {_ScaleWidget as ScaleWidget} from '@deck.gl/widgets';
+import {Deck} from '@deck.gl/core';
+
+const deck = new Deck({
+  widgets: [new ScaleWidget({placement: 'bottom-left', label: 'Scale'})]
+});
+```
+
+## Props
+
+#### `id` (string, optional) {#id}
+
+Default: `'scale'`
+
+Unique identifier for the widget.
+
+#### `placement` (string, optional) {#placement}
+
+Default: `'bottom-left'`
+
+Widget position within the view. Valid options are `top-left`, `top-right`, `bottom-left`, `bottom-right`, or `fill`.
+
+#### `label` (string, optional) {#label}
+
+Default: `'Scale'`
+
+Tooltip label for the widget.
+
+#### `onCapture` (Function, optional) {#oncapture}
+
+`(widget: ScaleWidget) => void`
+
+Callback when the widget is clicked.
+
+#### `style` (object, optional) {#style}
+
+Default: `{}`
+
+Additional CSS styles for the widget. Supports camelCase and kebab-case CSS variables.
+
+#### `className` (string, optional) {#classname}
+
+Default: None
+
+Custom class name for the widget element.
```

Comment:
Now that we have the base Widget class we could consolidate these standard props into a Widget class doc, similar to Layer.